docker-assets/run-postgresql

```diff
@@ -12,6 +12,24 @@ function set_miq_role() {
   psql --command "ALTER ROLE \"${POSTGRESQL_USER}\" SUPERUSER LOGIN PASSWORD '${POSTGRESQL_PASSWORD}';"
 }
 
+# Include MIQ config overrides if directory present
+function include_miq_config() {
+  # Set and create default dir if POSTGRESQL_CONFIG_DIR is not defined, compatible with older builds
+  POSTGRESQL_CONFIG_DIR=${POSTGRESQL_CONFIG_DIR:-/var/lib/pgsql/conf.d}
+
+  # Check for POSTGRESQL_CONFIG_DIR presence in postgresql.conf, if already there skip.
+  [[ $(grep ${POSTGRESQL_CONFIG_DIR} ${PGDATA}/postgresql.conf) ]] && RET=$? || RET=$?
+  if [[ $RET == "0" ]]; then
```

Comment:
I tried to avoid checking straight on $? , too many times things get in the middle by accident. That's why I chose to save status on a VAR. Script is run with set -eu and is very picky about statuses.